src/components/FilterEditor.vue

```diff
@@ -77,7 +78,8 @@ export default class FilterEditor extends Vue {
   readonly defaultValue = DEFAULT_FILTER;
 
   get conditionsTree() {
-    return buildConditionsEditorTree(castFilterStepTreeValue(this.filterTree, this.columnTypes));
+    const esJsonEnabled = this.$store.state.vqb.feature_flags.QUERYBUILDER_ESJSON == 'enabled'
```

Comment:
Nice introduction of feature flags!
In JS/TS we prefer camelCase everywhere (you imported a python convention here :p)